lib/rucio/core/transfer.py

```diff
@@ -1448,10 +1448,13 @@ def __load_distance_edges_node(rse_id, session=None):
         for distance in session.query(models.Distance).join(models.RSE, models.RSE.id == models.Distance.dest_rse_id)\
                                .filter(models.Distance.src_rse_id == rse_id)\
                                .filter(models.RSE.deleted == false()).all():
+            ranking = None
```

Comment:
does this make sense? Is the algorithm okay with `None` values in the distance_graph?

Reply:
It is covered by the algorithm which is just skipping these entries. But it is a good point actually, it is better to just skip `None` entries here entirely, so they don't even have to be iterated in the algorithm later.